docs/sources/flow/getting-started/configure-agent-clustering.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Configure agent clustering with Helm
+weight: 400
+---
+
+# Configure agent clustering with Helm
```

Comment:
@clayton-cornell I imagine this page will have more than just Helm instructions in the future, but I'm not sure if we should start out with having Helm be an `h2`. 

(Also: each deployment environment will have different "before you begin" requirements, since Helm has the unique requirement of a StatefulSet.

WDYT?

Reply:
@rfratto Yes, we could simplify the heading to be more generic and bump the Helm Chart up one level.

```
# Configure Grafana Agent clustering

Grafana Agent can be configured to run with [clustering][] so that
individual agents can work together for workload distribution and high
availability.

(and any other intro text)

## Configure Grafana Agent clustering with Helm Chart

(Instructions specific to Helm)
````

This leaves it open/easy to add any future cluster config tooling/options.

Reply:
Done, let me know what you think :)